dir.props

```diff
@@ -251,7 +251,7 @@
     <UAPAOTPackageRuntimePath>$(BinDir)pkg\uapaot\lib</UAPAOTPackageRuntimePath>
 
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
-    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp'">true</BinPlaceTestSharedFramework>
+    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp' or '$(_bc_TargetGroup)' == 'uap'">true</BinPlaceTestSharedFramework>
```

Comment:
If we do decide to keep binplacing the runtime dir, I would instead have create a property specific to UAP as opposed to setting this one. Something like `<BinPlaceUAPTestRuntime>`